app/views/document_lead_image/_image_list.html.erb

```diff
@@ -0,0 +1,38 @@
+<table>
+  <% @document.images.each_with_index do |image, i| %>
+    <tr>
+      <td class="image-list">
+        <%= image_tag(url_for(image.thumbnail), id: "image-#{i}") %>
```

Comment:
Could we use a component here instead?

https://govuk-publishing-components.herokuapp.com/component-guide/image_card

Reply:
I'm thinking the whole component could be inside a form to select it as the lead image, with a special 'no lead image' one above the rest, to replace the remove button - it's a bit weird having that on all of the images.

Reply:
Not sure I understand how the form is submitted without a button?

Reply:
Oh yeah, that's true. Perhaps not then. We could still put it alongside and use the component - I was thinking it would get us to preview earlier. But you're right it's not part of this PR.

Reply:
That said, it seems silly to right a whole new component if we could just tweak the design a bit. Perhaps @soniaturcotte (hello again ;-) could comment on this?